packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -442,6 +442,22 @@ class Auth {
         }
     }
 
+    logTokenRequestError = async (error: Error) => {
+        // the regular error.message will return only the generic status code message
+        // ie. 'Bad Request' for 400. We need to drill specifically into the ResponseError
+        // to get a more descriptive error message from SLAS
+        if (error instanceof Error && 'response' in error) {
+            // commerce-sdk-isomorphic throws a `ResponseError`, but doesn't export the class.
+            // We can't use `instanceof`, so instead we just check for the `response` property
+            // and assume it is a fetch Response.
+            const json = await (error['response'] as Response).json()
+            const status_code: string = json.status_code
+            const responseMessage: string = json.message
+
+            this.logger.error(`${status_code} ${responseMessage}`)
```

Comment:
Similar to the previous comment, do we really need to log the response details?

I think as long as we handle _any_ refresh token errors (rather than one with a specific error message), then this is already better error handling than before.

Reply:
I think we **do** need to log the response details because there are a variety of different types of problems that SLAS could respond with and we don't want this to happen in a place where developers can't get into our SDK to get those clarifying response error messages that are missing

In "development mode" they can look at the Network tab in Chrome, but if a customer gets the issue and can't reproduce it, seeing the verbose message would become critical in catching and fixing errors in production